source/models/relationships.md

```diff
@@ -174,9 +174,9 @@ import PaymentMethod from './payment-method';
 
 export default PaymentMethod.extend({
   last4: DS.attr(),
-  
+
   obfuscatedIdentifier: computed('last4', function () {
-    return `**** **** **** ${this.get('last4')}`;
+    return `**** **** **** ${this.last4}`;
```

Comment:
I'm unsure about this one as it involves Ember Data models. I think we might have to keep using `get()` here for now. 🤔

Reply:
I believe as long as `last4` isn't a relationship we should be fine, because that's where the proxies are.